xml/System/Type.xml

```diff
@@ -4961,7 +4961,7 @@ The current instance or <paramref name="interfaceType" /> argument is an open ge
 ## Remarks  
  Members include properties, methods, fields, events, and so on.  
   
- The <xref:System.Type.GetMember%2A> method does not return members in a particular order, such as alphabetical or declaration order. Your code must not depend on the order in which members are returned, because that order varies.  
+ In .NET 6 and below versions the <xref:System.Type.GetMember%2A> method does not return members in a particular order, such as alphabetical or declaration order. Your code must not depend on the order in which members are returned, because that order varies. However, starting with .NET 7, the ordering is deterministic based upon the metadata ordering in the assembly.
```

Comment:
```suggestion
 In .NET 6 and earlier versions, the <xref:System.Type.GetMember%2A> method does not return members in a particular order, such as alphabetical or declaration order. Your code must not depend on the order in which members are returned, because that order varies. However, starting with .NET 7, the ordering is deterministic based upon the metadata ordering in the assembly.
```